api/swagger.yaml

```diff
@@ -3344,7 +3344,7 @@ paths:
     get:
       summary: "Get container stats based on resource usage"
       description: |
-        This endpoint returns a live stream of a container’s resource usage statistics.
+        This endpoint returns a live stream of a container's resource usage statistics.
```

Comment:
These don't have to be changed, see http://practicaltypography.com/straight-and-curly-quotes.html